### PR TITLE
Add API call "editMonitor"

### PR DESCRIPTION
**Pull request by chemelli74**
## Proposed change

Add API call "editMonitor"

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (which adds functionality)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] Local tests pass.
- [x] There is no commented out code in this PR.
- [x] The code has been formatted using Black (`make lint`)
- [x] Coverage is still 100% (check with `make coverage`)
- [ ] Tests have been added to verify that the new code works.

### CODE REVIEW

**Review by ludeeus — CHANGES_REQUESTED** [review]
This is missing tests.

**Comment by chemelli74** [comment]
> This is missing tests.

Tests added.

File: pyuptimerobot/uptimerobot.py [hunk 1]
```diff
@@ -20,3 +20,7 @@ async def async_get_monitors(self, **kwargs) -> UptimeRobotApiResponse:
     @api_request("/getAccountDetails")
     async def async_get_account_details(self, **kwargs) -> UptimeRobotApiResponse:
         """Get account details from API."""
+
+    @api_request("/editMonitor")
+    async def async_edit_monitor(self, **kwargs) -> UptimeRobotApiResponse:
```
Comment (ludeeus) [inline pyuptimerobot/uptimerobot.py:25]: This needs more work, the `id` argument is not optional.

**Review by ludeeus — CHANGES_REQUESTED** [review]
This is missing tests

File: pyuptimerobot/uptimerobot.py (hunk 1 above)
Comment (chemelli74) [reply pyuptimerobot/uptimerobot.py:25]: Unfortunately I have not much experience with decorators :-(

Comment (ludeeus) [reply pyuptimerobot/uptimerobot.py:25]: This is not about decorators, this is about the signature of the method.

File: pyuptimerobot/uptimerobot.py [hunk 2]
```diff
@@ -22,5 +22,7 @@ async def async_get_account_details(self, **kwargs) -> UptimeRobotApiResponse:
         """Get account details from API."""
 
     @api_request("/editMonitor")
-    async def async_edit_monitor(self, **kwargs) -> UptimeRobotApiResponse:
+    async def async_edit_monitor(
+        self, monitor_id: int, **kwargs
```
Comment (ludeeus) [inline pyuptimerobot/uptimerobot.py:26]:
```suggestion
        self,
        id: int,
        **kwargs,
```
`monitor_id` and `id` is not the same

**Review by ludeeus — CHANGES_REQUESTED** [review]

**Pull request merged**